### PR TITLE
[ORCA-112] Added content for ORCA Ingest Workflow

### PR DESCRIPTION
**Pull request by Quarriers89**
## Summary of Changes

Added steps for the Ingest Workflow and details on the MoveGranuleStep. Made a couple of corrections from other DAAC input.

Addresses [ORCA-112: Create "ORCA Archive Workflow" page for ORCA developer documentation.](https://bugs.earthdata.nasa.gov/browse/ORCA-112)

## Changes

* Added steps for the Ingest Workflow
* Added section to detail the MoveGranuleStep
* ...

## Type of change

Please delete options that are not relevant.

- [X] This change requires a documentation update

## How Has This Been Tested?

N/A

## Checklist:

- [X] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [X] User documentation
 - [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
    - [ ] Unit tests
    - [ ] Integration tests
    - [ ] Manual tests (outlined above)
- [ ] New and existing unit tests pass locally with my changes
    - [ ] Automated tests passing (if not fork)
    - [ ] Manual testing/integration testing passes (if forked)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code has passed security scanning
    - [ ] Snyk
    - [ ] git-secrets

## What is the current behavior?

N/A

## What is the expected correct/added behavior?

N/A

## Relevant logs and/or screenshots

N/A

## Possible fixes

N/A

### CODE REVIEW

**Comment by Quarriers89** [comment]
Initial PR was in the Master branch, where comments were added. Addressed all comments from the Master branch PR, closed, and opened this PR in the Develop branch.

**Comment by AndrewDorn** [comment]
Looks good to me.

**Comment by AndrewDorn** [comment]
Still good.

**Comment by Quarriers89** [comment]
Reviewed by team. OK to close.

File: README.md [hunk 1]
```diff
@@ -259,7 +263,9 @@ variable "database_app_user_pw" {
 The values corresponding to these variables must be set in your `cumulus-tf/terraform.tfvars` file, but note that many of these variables are actually hardcoded at the time of updating this README
 
 #### Adding the Copy To Glacier Step to the Ingest Workflow
-Navigate to `cumulus-tf/ingest_granule_workflow.tf` then add the following step after the PostToCMR step being sure to change the PostToCMR's "Next" parameter equal to "CopyToGlacier"
+Navigate to `cumulus-tf/ingest_granule_workflow.tf` then add the following
+step to anywhere after the MoveGranule step being sure to change the 
+PostToCMR's "Next" parameter equal to "CopyToGlacier".
```
Comment (bhazuka) [inline README.md:268]: Remove `PostToCMR's`

File: website/docs/about/introduction/glossary.md [hunk 2]
```diff
@@ -152,6 +152,10 @@ Refers to a [terraform module](https://www.terraform.io/docs/configuration/modul
 
 
 ## N
+### NGAP
+NASA General Application Platform. NGAP provides a cloud-based
+Platform-as-a-Service (PaaS) and Infrastructure-as-a-Service (IaaS) for
+ESDIS applications.
```
Comment (bhazuka) [inline website/docs/about/introduction/glossary.md:158]: Not sure if it matters or if we should add that it also provides our ATO to operate in the AWS commercial cloud.

File: website/docs/developer/deployment-guide/deployment-with-cumulus.md [hunk 3]
```diff
@@ -648,7 +724,7 @@ run `terraform destroy`.
 To configure a collection to enable ORCA, add the line
 `"granuleRecoveryWorkflow": "DrRecoveryWorkflow"` to the collection configuration
 as seen below. Optionally, you can exclude files by adding values to an
-`"exclue_file_type"` variable. For more information, see the documentation on the
+`"excludeFfileType"` variable. For more information, see the documentation on the
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-with-cumulus.md:727]: typo `excludeFfileType` should be `excludeFileTypes`

**Review by bhazuka — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 ## ORCA Static Documentation
 
+ORCA documentation can be found at nasa.github.io/cumulus-orca. The
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply README.md:3]: Added link for nasa.github.io/cumulus-orca.

File: README.md (hunk 1 above)
Comment (Quarriers89) [reply README.md:268]: Removed PostToCMR's.

File: website/docs/about/introduction/glossary.md (hunk 2 above)
Comment (Quarriers89) [reply website/docs/about/introduction/glossary.md:158]: Added content into the NGAP definition for the ATO.

File: website/docs/developer/deployment-guide/deployment-with-cumulus.md (hunk 3 above)
Comment (Quarriers89) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:727]: Edited typo to excludeFileTypes.

**Review by bhazuka — APPROVED** [review]

**Pull request merged**